src/main/java/com/atlassian/db/replica/api/AuroraMultiReplicaConsistency.java

```diff
@@ -47,8 +55,17 @@ public Builder auroraConnectionDetails(AuroraConnectionDetails auroraConnectionD
             return this;
         }
 
+        public Builder discoveredReplicasCache(SuppliedCache<Collection<Database>> discoveredReplicasCache) {
```

Comment:
It seems to conflict with (https://github.com/atlassian-labs/db-replica/pull/118). CC: @pjaronski-atlassian

Reply:
It's not in conflict with @pjaronski-atlassian changes. Piotr replaces AuroraConnectionDetails with connection pool provider. I'm adding additional cache for discoverer.

Reply:
`Database` is internal. Isn't this an implementation detail leak?